resources/public/api/conf/2.0/employments_financial_details_retrieve.yaml

```diff
@@ -71,6 +72,8 @@ responses:
             $ref: './common/errors.yaml#/components/examples/ruleTaxYearRangeInvalid'
           RULE_TAX_YEAR_NOT_SUPPORTED:
             $ref: './common/errors.yaml#/components/examples/ruleTaxYearNotSupported'
+          RULE_OUTSIDE_AMENDMENT_WINDOW:
```

Comment:
This is the wrong end point. Looks like it was missed for Amend a Custom employment